spec/classes/data_types_spec.rb

```diff
@@ -174,7 +182,15 @@
             context "when #{var_name} (#{type}) is set to valid #{valid} (as #{valid.class})" do
               let(:params) { [mandatory_params, var[:params], { "#{var_name}": valid, }].reduce(:merge) }
 
-              it { is_expected.to compile }
+              it do
```

Comment:
We probably want to have tests as well on what happens if we do not get a proper answer from function.

Reply:
Maybe this is applicable in a different spec file though. Maybe these tests already exists? Don't remember :P

Reply:
yes, these use cases are included in spec/classes/init_spec.rb